modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, you should:
+
+* Have access to the cluster as a user with `cluster-admin` privileges.
+* Have a recent etcd backup in case your update fails and you must restore your cluster to a previous state.
+* You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
```

Comment:
I'd move this to prereqs for update modules (and ensure parallel wording with other prereqs)